DS4Windows/DS4Library/InputDevices/JoyConDevice.cs

```diff
@@ -1252,6 +1252,8 @@ public void SetInitRumble()
         private byte[] Subcommand(byte subcommand, byte[] tmpBuffer, uint bufLen,
             bool checkResponse = false)
         {
+            int retryLimit = 100; stickCalibRetryTag:
```

Comment:
Is 100 retry limit really needed (and will you ever actually read the data from the controller if you don't read it the first time anyway)?

Reply:
100 was chosen arbitrarily as a sufficiently large value. Based on tests, when this bug occurs, the 3rd-party Joycon typically requires 2-10 retries, while the official Joycon only needs 1-3 retries.